conda-forge.yml

```diff
@@ -1,9 +1,15 @@
-build_platform: {osx_arm64: osx_64}
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
The default is now to emulate for other Linux architectures on Azure. However this can be quite slow in terms of build time. Have configured this to use cross-compilation, which appears to be working